docs/modules/ROOT/pages/reference/extensions/jasypt.adoc

```diff
@@ -89,7 +89,7 @@ TIP: You can use the ability to mask security sensitive configuration in Camel b
 You can also disable the startup configuration summary with the configuration `camel.main.autoConfigurationLogSummary = false`.
 
 [id="extensions-jasypt-usage-injecting-encrypted-configuration"]
-==== Injecting encrypted configuration
+=== Injecting encrypted configuration
```

Comment:
I feel like it should be even one level lower, `Injecting encrupted configuration` sounds unrelated to `Maven coordinates`

Reply:
forget about it, I missed the `usage` level